app/models/user.rb

```diff
@@ -7,6 +7,10 @@ class User < ApplicationRecord
 
   validates_presence_of :first_name, :last_name
 
+  def full_name
+    "#{first_name} #{last_name}"
+  end
+
```

Comment:
See remark above. This has already been provided for in the `MembershipApplicationsHelper` as `member_full_name`